NEWS.md

```diff
@@ -55,22 +51,43 @@ install.packages("polars0", repos = "https://rpolars.r-universe.dev")
   #> Run `rlang::last_trace()` to see where the error occurred.
   ```
 
-* Conversion from polars objects to R vectors has been revamped: `$to_r()`,
-  `$to_list()` and `$to_data_frame()` no longer exist. Instead, you must use
-  `as.data.frame()`, `as.list()` and `as.vector()`.
+- Conversion from polars objects to R vectors has been revamped: `<series>$to_r()`,
+  `<series>$to_list()` and `<dataframe>$to_data_frame()` no longer exist. Instead, you must use
+  `as.data.frame(<dataframe>)`, `as.list(<dataframe>)`, `as.vector(<series>)`, or `<series>$to_r_vector()`.
 
-  `as.vector()` will remove attributes that might be useful, for instance to
+  `as.vector(<series>)` will remove attributes that might be useful, for instance to
   convert Int64 values using the bit64 package or to convert Time values using
-  the hms package. If finer control is needed, use either `$to_r_vector()` or
-  set options for the entire session with `polars_options()`.
+  the hms package. If finer control is needed, use either `<series>$to_r_vector()`.
 
-* In general, polars now uses dots (`...`) in two scenarios:
+  ```r
+  s_int64 <- as_polars_series(1:5)$cast(pl$Int64)
 
-  1. to pass an unlimited number of inputs (for instance in `select()`, `cast()`,
-     or `group_by()`), using [dynamic-dots](https://rlang.r-lib.org/reference/dyn-dots.html).
+  withr::with_options(
+    list(polars.to_r_vector.int64 = "integer64"),
+    as.vector(s_int64)
+  )
+  #> `int64` is overridden by the option "polars.to_r_vector.int64" with the string "integer64"
+  #> ℹ `as.vector()` on a Polars Series of type i64 may drop some useful attributes.
+  #> ℹ Use `$to_r_vector()` instead for finer control of the conversion from Polars to R.
+  #> [1] 4.940656e-324 9.881313e-324 1.482197e-323 1.976263e-323 2.470328e-323
+
+  withr::with_options(
+    list(polars.to_r_vector.int64 = "integer64"),
+    s_int64$to_r_vector()
+  )
```

Comment:
This only shows setting global options. Can we have an example with using arguments of `$to_r_vector()`?